url2df0f7c811_przemekgalazka_legacycode_tgz-pJ8-pl_cc_core_cmd_CommandJ8/src/pl/cc/utils/Utils.java

```diff
@@ -0,0 +1,100 @@
+package pl.cc.utils;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+import org.apache.log4j.Logger;
+
+/**
+ * Rďż˝ne poďż˝yteczne
+ * 
+ * @since 2007-06-26
+ */
+public class Utils {
+	public static final Logger log = Logger.getLogger(Utils.class);
+	//public static final String NULL = "- null -";
+	public static final String NULL = "";
+	
+	public static String getNumber(String s){
+		//+INFO New Call from '977', agent: 'Agent/301', queue: 'kolejka'
+		int a = s.indexOf("'");
+		int b = s.indexOf("'", a+1);
+		int c = s.indexOf("'", b+1);
+		int d = s.indexOf("'", c+1);
+		int e = s.indexOf("'", d+1);
+		int f = s.indexOf("'", e+1);
+		return s.substring(e+1,f)+" "+s.substring(a+1,b);
+	}
+
+    public static String[] getInfo(String s){
+        //+INFO New Call from '977', agent: 'Agent/301', queue: 'kolejka'
+        int a = s.indexOf("'");
+        int b = s.indexOf("'", a+1);
+        int c = s.indexOf("'", b+1);
+        int d = s.indexOf("'", c+1);
+        int e = s.indexOf("'", d+1);
+        int f = s.indexOf("'", e+1);
+        return new String[] {s.substring(e+1,f),s.substring(a+1,b)};
+    }
+
+
+    public static Date getNow(){
+		GregorianCalendar gc = new GregorianCalendar();
+		return gc.getTime();
+	}
+	
+	/**
+	 * @return obecna data w formacie "HH:mm:ss"
+	 */
+	public static String getNowFormated(){
+		GregorianCalendar gc = new GregorianCalendar();
+		SimpleDateFormat dateFormat = new SimpleDateFormat("HH:mm:ss");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you do not call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.